engine/remote-components/src/main/java/org/datacleaner/components/remote/RemoteTransformer.java

```diff
@@ -313,7 +313,7 @@ public void map(BatchSource<InputRow> source, BatchSink<Collection<Object[]>> si
         } catch (RuntimeException e) {
             boolean alreadyFailed = failed.getAndSet(true);
             if (!alreadyFailed) {
-                throw e;
+                throw new RuntimeException("Remote transformer failed: " + e.getMessage(), e);
```

Comment:
This seems like a step back? I would prefer getting the cause of the error thrown rather than a wrapped exception (unless that wrapped exception is able to provide better details, which it doesn't look like).